topics/wordplate/index.md

```diff
@@ -0,0 +1,11 @@
+---
+topic: wordplate
+created_by: Vincent Klaiber
+display_name: WordPlate
+github_url: https://github.com/wordplate
+logo: wordplate.png
+released: Mars 18, 2015
+short_description: WordPlate is a modern WordPress stack built with Composer.
+url: https://wordplate.github.io/
+---
+WordPlate is a modern WordPress stack built with Composer.
```

Comment:
Looks like you have a failing test from the body and the short_description being the same. Can you expand this line to give more details about WordPlate?

Reply:
Done!